src/features/non-tracking-3p-cookies.js

```diff
@@ -0,0 +1,39 @@
+import { defineProperty, postDebugMessage } from '../utils'
+
+function blockCookies (debug) {
+    // disable setting cookies
+    defineProperty(globalThis.document, 'cookie', {
+        configurable: false,
+        set: function (value) {
+            if (debug) {
+                postDebugMessage('jscookie', {
+                    action: 'block',
+                    reason: 'tracker frame',
+                    documentUrl: globalThis.document.location.href,
+                    scriptOrigins: [],
+                    value: value
+                })
+            }
+        },
+        get: () => {
+            if (debug) {
+                postDebugMessage('jscookie', {
+                    action: 'block',
+                    reason: 'tracker frame',
+                    documentUrl: globalThis.document.location.href,
+                    scriptOrigins: [],
+                    value: 'getter'
+                })
+            }
+            return ''
+        }
+    })
+}
+
+export function init (args) {
+    args.cookie.debug = args.debug
+    if (globalThis.top !== globalThis && !args.cookie.isTrackerFrame && args.cookie.shouldBlock && args.cookie.isThirdParty) {
+        // overrides expiry policy with blocking - only in subframes
+        blockCookies(args.debug)
+    }
+}
```

Comment:
This feels weird to me as the script is exactly the same as our `tracking-cookies-3p` script with the exception of the `isTrackingFrame` check. Although with the architecture of the content-scope-scripts I'm not sure if there is a more DRY approach.